package.json

```diff
@@ -40,6 +40,7 @@
     "html-to-text": "2.1.3",
     "js-yaml": "3.6.1",
     "junk": "1.0.3",
+    "lodash.defaultsdeep": "4.5.1",
```

Comment:
I don't know if it makes sense to use full `lodash`, we may need more utilities in future.